main.py

```diff
@@ -3,14 +3,32 @@
 # throughout this file
 import pygame
 
+from constants import SCREEN_HEIGHT, SCREEN_WIDTH
+
 
 def main():
-    try:
-        pygame.init()
-        print('Starting asteroids!')
-        pygame.quit()
-    except Exception as e:
-        print(f'Failed to initialize Pygame: {e}')
+    pygame.init()  # Initialize Pygame
+
+    # Set up the display
+    screen = pygame.display.set_mode((SCREEN_WIDTH, SCREEN_HEIGHT))
+    
+    # Start the game loop
+    clock = pygame.time.Clock()
+    while True:
+        clock.tick(60) 
+        # Event handling to check if the user wants to quit
+        for event in pygame.event.get():
+            if event.type == pygame.QUIT:
+                pygame.quit()
+                return
+            elif event.type == pygame.KEYDOWN:
+
+        # Fill the screen with black color (RGB: 0, 0, 0)
```

Comment:
**Suggestion:** Ensure consistent frame rate in the game loop for better gameplay experience.

**Existing Code:**
```Python
screen.fill((0, 0, 0))
```

**Improved Code:**
```Python
clock.tick(60)  # Limit to 60 frames per second
screen.fill((0, 0, 0))
```

**Details:**
Ensure that the game loop has a consistent frame rate to improve gameplay experience.